_opcodes/op090.html

```diff
@@ -5,7 +5,7 @@
 param2: "Type"
 bg1: 1
 bg2: 1
-bgee: 1
```

Comment:
These three thief skills one can also be merged back — just make the note explicitly mention it's for the EEs, since the rest don't have those tables.